test-data/unit/check-isinstance.test

```diff
@@ -300,6 +307,7 @@ class B(A):
     b = 1
 
 x = A()
+def f(): x  # Prevent redefinition
```

Comment:
I would create a full list of "hack" patterns used, and place it in one of the test files as a comment.

Reply:
I'll add a separate file where I describe them, as these aren't specific to any particular test file.